slack_handlers/handlers.py

```diff
@@ -747,6 +750,236 @@ def handle_list_team_links(say, user):
     )
 
 
+# Helper function to handle ROTA operations
+@command_meta(
+    name="rota",
+    description="Manage release rotation assignments in Google Sheets",
+    arguments={
+        "action": {
+            "description": "Action to perform",
+            "required": True,
+            "type": "str",
+            "choices": ["add", "check", "replace"],
+        },
+        "release": {
+            "description": "Release version (e.g., 4.15.1)",
+            "required": False,
+            "type": "str",
+        },
+        "start_date": {
+            "description": "Start date in YYYY-MM-DD format (must be a Monday)",
+            "required": False,
+            "type": "str",
+        },
+        "end_date": {
+            "description": "End date in YYYY-MM-DD format (must be a Friday)",
+            "required": False,
+            "type": "str",
+        },
+        "pm": {
+            "description": "Project Manager username",
+            "required": False,
+            "type": "str",
+        },
+        "qe1": {
+            "description": "Primary QE engineer username",
```

Comment:
```suggestion
            "description": "Primary QE username",
```
QE is quality engineer